hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -14,6 +14,7 @@ import {
     TableCell,
 } from "@material-ui/core";
 import Typography from "@material-ui/core/Typography";
+import TeamCheckoutOrderTable from "../../components/teamDetail/TeamCheckoutOrderTable/TeamCheckoutOrderTable";
```

Comment:
use absolute import here.  If you want pycharm to do this automatically go to Settings -> Editor -> Code Style -> JavaScript and you should be able to uncheck relative path imports